packages/manager/apps/hub-next/src/i18n.js

```diff
@@ -0,0 +1,44 @@
+import { nextTick } from 'vue';
+
+/* eslint-disable no-param-reassign */
+export function setI18nLanguage(i18n, locale) {
+  if (i18n.mode === 'legacy') {
+    i18n.global.locale = locale;
+  } else {
+    i18n.global.locale.value = locale;
+  }
```

Comment:
Is this some i18n library recommandation ?

Reply:
Yes, just picked up the code they're recommending from their docs :  https://vue-i18n.intlify.dev/guide/advanced/lazy.html

Reply:
> If you specified `legacy: true` option in `createI18n`, return `legacy`, else `composition`
https://vue-i18n.intlify.dev/api/general.html#mode

> In Vue I18n v9 and later, the API offered by Vue I18n v8.x is called Legacy API mode.

It could impact the way you access the VueI18n instance (example: https://vue-i18n.intlify.dev/api/general.html#global)

Here, we are using composition mode, so I think we could remove the condition.